modules/apps/transfer/keeper/relay.go

```diff
@@ -430,7 +434,7 @@ func createPacketDataBytesFromVersion(appVersion, sender, receiver, memo string,
 	case types.V1:
 		// Sanity check, tokens must always be of length 1 if using app version V1.
 		if len(tokens) != 1 {
-			return nil, errorsmod.Wrapf(ibcerrors.ErrInvalidRequest, "cannot transfer multiple coins with %s", types.V1)
+			return nil, errorsmod.Wrapf(types.ErrInvalidVersion, "length of tokens must be equal to 1 if using %s version", types.V1)
```

Comment:
would recommend doing changes like this against main since it is unrelated to the port router changes

Reply:
this was the result of a weird merge, because we used to `panic` here so I added a similar change but not identical, will revert